attributes/default.rb

```diff
@@ -3,3 +3,9 @@
 default['snapraid']['data_disks'] = ['/media/disk1', '/media/disk2', '/media/disk3', '/media/disk4']
 default['snapraid']['parity_disks'] = ['/media/p-disk1', '/media/p-disk2']
 default['snapraid']['content_files'] = ['/var/snapraid/content', '/var/disk1/content', '/var/disk2/content']
+
+# Version options
+default['snapraid']['version']['number'] = '7.1'
```

Comment:
Looks like it's only went to RC 3 days ago though http://sourceforge.net/p/snapraid/discussion/1677233/thread/ab06c005/
I'll update to 8.0 once it's released if that's okay by you?